guides/release/templates/handlebars-basics.md

```diff
@@ -1,85 +1,203 @@
-Ember uses a templating language based on [Handlebars templating library](http://www.handlebarsjs.com) to power your app's user interface.
-Ember templates contain static HTML and dynamic content inside Handlebars expressions, which are invoked with double curly braces: `{{}}`.
+Templates are the home for what the user sees, like forms, buttons, links, and headings.
 
+In this section of the Guides, you will learn about where to write HTML markup, plus how to add interaction, dynamically changing content, styling, and more. If you want to learn in a step-by-step way, you should begin your journey in the [Tutorial]() instead.
 
-### Displaying Properties
+## Writing plain HTML
 
-Templates are backed with a context. A context is an object from which
-Handlebars expressions read their properties. In Ember this is often a component. For templates rendered by a route (like `application.hbs`), the context is a controller.
+Ember templates have some superpowers, but let's start with regular HTML. For any file in an Ember app that has an extension ending in `.hbs`, you can write HTML markup in it as if it was an `.html` file. HTML is the language that browsers understand for laying out content on a web page. `.hbs` stands for Handlebars, the name of a tool that lets you write more than just HTML in your templates.
 
-For example, this `application.hbs` template will render a first and last name:
+For example, every Ember app has a file called `application.hbs`. You can write regular HTML markup there or in any other `hbs` file:
 
-```handlebars {data-filename=app/templates/application.hbs}
-Hello, <strong>{{this.firstName}} {{this.lastName}}</strong>!
+```hbs {data-filename=app/templates/application.hbs data-update=false}
+<h1>Starting simple</h1>
+<p>
+  This is regular html markup inside an hbs file
+</p>
+```
+
+If you make a mistake like forgetting to close a tag or missing a quotation mark, you will see some errors when you run the app with `ember serve`. However, reading the error message on the page or in your browser's developer console will get you going again right away.
```

Comment:
we could phrase this as the developer having a sidekick that helps them write better HTML. "the compiler is your friend" sort of thing.

Reply:
love this, done